onboarding/source/controller.c

```diff
@@ -94,6 +120,9 @@ static void ledTimerCallback(TimerHandle_t xTimer) {
 static void lightTimerCallback(TimerHandle_t xTimer) {
     /* USER CODE BEGIN */
     // Send light event to light service queue
+    ASSERT(xTimer != NULL);
+
+    sendToLightServiceQueue(MEASURE_LIGHT);
```

Comment:
Deal with the case where the event isn't sent to the queue. Printing a message is good enough for this challenge.

Reply:
Also, you should be passing in a pointer to the event. So have something like `light_event_t event = MEASURE_LIGHT;` and then pass in `&event`.